src/hmem.c

```diff
@@ -273,7 +273,8 @@ static ssize_t ofi_copy_mr_iov(struct ofi_mr **mr, const struct iovec *iov,
 		size_t size, int dir)
 {
 	uint64_t done = 0, len;
-	uint64_t hmem_iface, hmem_device;
+	uint64_t hmem_iface = FI_HMEM_SYSTEM, hmem_device = 0, hmem_flags = 0;
+	void *hmem_data = NULL;
```

Comment:
you can drop the initialization here since you initialize it on the loop

Reply:
Yeah. That was my PTSD because of coverity.

I will remove it for prov/shm for coding consistency.